app/models/container_node.rb

```diff
@@ -88,11 +88,7 @@ def evaluate_alert(_alert_id, _event)
     true
   end
 
-  supports :external_logging do
-    unless ext_management_system.respond_to?(:external_logging_route_name)
```

Comment:
Aren't we losing this check with this change?

Reply:
There is no instance-level config so moving `supports :external_logging` to just the openshift class looks right to me